django/thunderstore/repository/mixins.py

```diff
@@ -1,7 +1,19 @@
+from thunderstore.community.models.community import Community
+
+
 class CommunityMixin:
     @property
     def community_identifier(self):
         return (
             self.kwargs.get("community_identifier", None)
             or self.request.community.identifier
         )
+
+    @property
+    def solved_community_from_identifier(self):
```

Comment:
Is there a reason this isn't simply called `community` (i.e. clashing with something)? Also this should be a cached property to avoid querying it multiple times.

Reply:
Named it this just for extra clarification, now that you mention it, better one could be thought of.